src/resultsdisassemblypage.cpp

```diff
@@ -45,12 +45,15 @@
 #include "parsers/perf/perfparser.h"
 #include "resultsutil.h"
 
+#include "models/filterandzoomstack.h"
 #include "models/costdelegate.h"
 #include "models/hashmodel.h"
 #include "models/topproxy.h"
 #include "models/treemodel.h"
 
 #include <QStandardItemModel>
+#include <QWheelEvent>
```

Comment:
please remove this feature, it should be handled generically - ideally through reusing KTextEditor for showing the assembly data

Reply:
this would also get us proper highlighting etc. pp.